### PR TITLE
styles.less examples

### PR DESCRIPTION
**Pull request by simurai**
This PR adds some examples (commented). Fixes https://github.com/atom/welcome/issues/34

It also changes the line about Less and mentions that you can use normal CSS (or Less). Some people might get discouraged when they read about having to learn Less.

### CODE REVIEW

File: dot-atom/styles.less [hunk 1]
```diff
@@ -4,22 +4,21 @@
  * This stylesheet is loaded when Atom starts up and is reloaded automatically
  * when it is changed.
  *
- * If you are unfamiliar with LESS, you can read more about it here:
- * http://www.lesscss.org
```
Comment (izuzak) [inline dot-atom/styles.less]: I kind of liked the link here which helps users learn more about less. If you don't feel strongly about removing this, could I talk you into keeping the link or adding some other links to point users to :books:?

**Comment by izuzak** [comment]
Other than that one comment/question: :+1: :zap:

File: dot-atom/styles.less (hunk 1 above)
Comment (simurai) [inline dot-atom/styles.less]: Yeah, sure. I rephrased it a bit and added the line with the link back.

File: dot-atom/styles.less [hunk 2]
```diff
@@ -4,22 +4,23 @@
...
+ * Uncomment the examples below to try it out. Or add your own CSS or Less.
```
Comment (mnquintana) [inline dot-atom/styles.less]: > try it out

Try what out? Probably better to be more explicit here.

Comment (simurai) [inline dot-atom/styles.less]: Refers to "reloaded automatically", but I shuffled it around. Now it's under the "Examples".

Still not sure about `(To see them, uncomment and save)`.

**Comment by kevinsawicki** [comment]
:+1:

**Pull request merged**